tgui/packages/tgui/interfaces/PipeDispenser.js

```diff
@@ -0,0 +1,102 @@
+import { useBackend, useLocalState } from '../backend';
+import { Button, Section, Flex, Box, NumberInput } from '../components';
+import { Window } from '../layouts';
+
+export const PipeDispenser = (props, context) => {
+  const { act, data } = useBackend(context);
+  const disposalpipes = data.disposalpipes || [];
+  const [amount, setAmount] = useLocalState(context, 'amount', 1);
+  const {
+    dispenser_ready,
+    windowName,
+    mobile,
+    removing_pipe,
+    laying_pipe,
+    max_disposal_pipes,
+  } = data;
+  return (
+    <Window
+      title={windowName}
+      width="300"
+      height={mobile ? 330 : 235}
+      fontFamily="Consolas"
+      font-size="10pt">
+      <Window.Content scrollable>
+        <Section>
+          {disposalpipes.map(disposalpipe => {
+            return (
+              <Flex key={disposalpipe.disposaltype} justify="space-between" align="stretch" style={{ "border-bottom": "1px #555 solid" }}>
+                <Flex.Item direction="row">
+                  {disposalpipe.image && (
+                    <Box style={{ "overflow": "show", "height": "32px" }}>
+                      <img
+                        src={`data:image/png;base64,${disposalpipe.image}`}
+                      />
+                    </Box>)}
+                </Flex.Item>
+                <Flex.Item direction="row"
+                  grow style={{
+                    "display": "flex",
+                    "justify-content": "center",
+                    "flex-direction": "column",
+                  }}>
+                  <Box>
+                    <Box inline>
+                      {disposalpipe.disposaltype}
+                    </Box>
+                  </Box>
+                </Flex.Item>
```

Comment:
Not quite sure what the intent of the `style` stuff here is. Made me read the rest of this layout a bit. Not able to debug right this second, can take a look at probably tomorrow if you'd rather wait until then for me to spin this up and play with.

Reply:
It aligns the text next to the icon, so it doesn't look elevated
![image](https://user-images.githubusercontent.com/62359847/220503046-99b4e2c2-6306-418b-be77-00bda6e2982f.png)